packages/client/components/auth/src/flows/Form.tsx

```diff
@@ -81,7 +91,7 @@ export function Fields(props: FieldProps) {
 
   return (
     <For each={props.fields}>
-      {(field) => (
+      {(field) => (field != "invite" || inviteCodeNeeded) && (
```

Comment:
It's better to use https://docs.solidjs.com/concepts/control-flow/conditional-rendering#show here? SolidJS recommends their components for better performance and such.

Reply:
Will adapt to that recommendation later